compoments/report-repair/address.js

```diff
@@ -43,39 +43,39 @@ const Address = ({handleChange, values}) => {
 
   return <div className="govuk-grid-row" data-cy="SectionLoaded">
     <div className="govuk-grid-column-two-thirds">
-      <fieldset className="govuk-fieldset">
-        <legend className="govuk-fieldset__legend govuk-fieldset__legend--l">
-          <h1 className="govuk-fieldset__heading">Where is the repair located?</h1>
-        </legend>
-        <form action="">
-          <div className={state.error.msg ? 'govuk-form-group govuk-form-group--error' : 'govuk-form-group'}>
-            <span id={'address-error'}
-              className="govuk-error-message">
-              {state.error.msg}
-            </span>
-            <Select
-              input={{
-                name: 'address',
-                onChange: onChange
-              }}
-              meta={state.error}
-            >
-              <option value="null">
-                {found_addresses}
+      <h1 className="govuk-heading-xl">Where is the repair located?</h1>
```

Comment:
The title should be "Select an address"
```suggestion
      <h1 className="govuk-heading-xl">Select an address</h1>
```